custom/import_xml.php

```diff
@@ -121,71 +121,77 @@ function setInsurance($pid, $ainsurance, $asubscriber, $seq) {
   if ($olddata['squad'] && ! acl_check('squads', $olddata['squad']))
    die("You are not authorized to access this squad.");
 
-  newPatientData(
-   $olddata['id'],
-   $apatient['title'],
-   $apatient['fname'],
-   $apatient['lname'],
-   $apatient['mname'],
-   $apatient['sex'],
-   $apatient['dob'],
-   $apatient['street'],
-   $apatient['zip'],
-   $apatient['city'],
-   $apatient['state'],
-   $apatient['country'],
-   $apatient['ss'],
-   $apatient['occupation'],
-   $apatient['phone_home'],
-   $apatient['phone_biz'],
-   $apatient['phone_contact'],
-   $apatient['status'],
-   $apatient['contact_relationship'],
-   $apatient['referrer'],
-   $apatient['referrerID'],
-   $apatient['email'],
-   $apatient['language'],
-   $apatient['ethnoracial'],
-   $apatient['interpreter'],
-   $apatient['migrantseasonal'],
-   $apatient['family_size'],
-   $apatient['monthly_income'],
-   $apatient['homeless'],
-   fixDate($apatient['financial_review']),
-   $apatient['pubpid'],
-   $pid,
-   $olddata['providerID'],
-   $apatient['genericname1'],
-   $apatient['genericval1'],
-   $apatient['genericname2'],
-   $apatient['genericval2'],
-   $apatient['billing_note'],
-   $apatient['phone_cell'],
-   $apatient['hipaa_mail'],
-   $apatient['hipaa_voice'],
-   $olddata['squad']
-  );
-
-  newEmployerData(
-   $pid,
-   $aemployer['name'],
-   $aemployer['street'],
-   $aemployer['zip'],
-   $aemployer['city'],
-   $aemployer['state'],
-   $aemployer['country']
-  );
-
-  setInsurance($pid, $ainsurance, $asubscriber, '1');
-  setInsurance($pid, $ainsurance, $asubscriber, '2');
-  setInsurance($pid, $ainsurance, $asubscriber, '3');
-
-  echo "<html>\n<body>\n<script language='JavaScript'>\n";
-  if ($alertmsg) echo " alert('$alertmsg');\n";
-  echo " if (!opener.closed && opener.refreshme) opener.refreshme();\n";
-  echo " window.close();\n";
-  echo "</script>\n</body>\n</html>\n";
-  exit();
+  if(!$alertmsg)
+  {
+  	newPatientData(
+   		$olddata['id'],
+   		$apatient['title'],
+   		$apatient['fname'],
+   		$apatient['lname'],
+   		$apatient['mname'],
+   		$apatient['sex'],
+   		$apatient['dob'],
+   		$apatient['street'],
+   		$apatient['zip'],
+   		$apatient['city'],
+   		$apatient['state'],
+   		$apatient['country'],
+   		$apatient['ss'],
+   		$apatient['occupation'],
+   		$apatient['phone_home'],
+   		$apatient['phone_biz'],
+   		$apatient['phone_contact'],
+   		$apatient['status'],
+   		$apatient['contact_relationship'],
+   		$apatient['referrer'],
+   		$apatient['referrerID'],
+   		$apatient['email'],
+   		$apatient['language'],
+   		$apatient['ethnoracial'],
+   		$apatient['interpreter'],
+   		$apatient['migrantseasonal'],
+   		$apatient['family_size'],
+   		$apatient['monthly_income'],
+   		$apatient['homeless'],
+   		fixDate($apatient['financial_review']),
+   		$apatient['pubpid'],
+   		$pid,
+   		$olddata['providerID'],
+   		$apatient['genericname1'],
+   		$apatient['genericval1'],
```

Comment:
"generic" stuff was a pre- "billing note" field usage, and no longer has a place.  Other fields like social security number have no place in usage as an importer (or anything else in the medical industry).  SSN's are only useful in the medical industry for social security claims, and that is handled in a different way.  I suggest avoiding the use of it in a standard distribution in any case (not relevant to this commit I suppose).